arrays.js

```diff
@@ -0,0 +1,94 @@
+const getNthElement = (index, array) => {
+  return array[index];
+};
+
+const arrayToCSVString = (array) => {
+  return array.join();
+};
+
+const csvStringToArray = (string) => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.slice(4, 0, element);
+};
+
+const removeNthElement = (index, array) => {
+  return array.splice(index, 1);
+};
+
+const numbersToStrings = (numbers) => {
+  for (let i = 0; i < numbers.length; i++) {
+    numbers[i] = numbers[i].toString();
+  }
+  return numbers;
+};
+
+const uppercaseWordsInArray = (strings) => {
```

Comment:
`i` is okay to use in loops, but we prefer it when code is more explicit. Because you have an array of `strings` here, it's a bit more readable to use `string`. For example:
```
for (let string = 0; string < strings.length; string += 1) {
  // do stuff here
}
```

We also prefer `+= 1` to `++`